src/pages/overview/Other.tsx

```diff
@@ -0,0 +1,46 @@
+import i18n from '@dhis2/d2-i18n'
+import React from 'react'
+import { OVERVIEW_SECTIONS, SECTIONS_MAP } from '../../lib'
+import { SummaryCard, SummaryCardGroup } from './card'
+import { OverviewGroup, OverviewGroupSummary } from './group'
+
+const TITLE = OVERVIEW_SECTIONS.other.titlePlural
+
+export const OtherOverview = () => {
+    return (
+        <OverviewGroup title={TITLE}>
+            {/* <OverviewGroupSummary>
+                {i18n.t(
+                    'Placeholder for other'
```

Comment:
just leaving this here until we decide if want a subtitle for Other section / what it should be